<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QCWindow</class>
 <widget class="QDialog" name="QCWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>654</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>902</width>
    <height>654</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>902</width>
    <height>654</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>QuickChatter</string>
  </property>
  <property name="windowIcon">
   <iconset theme="applications-accessories"/>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>901</width>
     <height>641</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Type</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="sendButton">
      <property name="text">
       <string>Send!</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QComboBox" name="methodBox">
      <item>
       <property name="text">
        <string>LLM</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>RiveScript</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPlainTextEdit" name="convoBox">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
